docs/CHANGELOG.rst

```diff
@@ -42,10 +45,27 @@ Changed
 - **BACKWARD INCOMPATIBLE:** Count fragments (or templates) instead of reads by
   default in ``featureCounts`` process and ``BBDuk - STAR - featureCounts``
   pipeline. The change applies only to paired-end data.
+- **BACKWARD INCOMPATIBLE:** Change Docker image in ``alignment-subread``
+  from ``resolwebio/legacy:1.0.0`` with Subread (v1.5.1) to ``resolwebio/rnaseq:3.2.0``
+  with Subread (v1.6.0). ``--unique_reads`` option was removed and ``--multiMapping``
+  option was added. By default aligner report uniquely mapped reads only.
 - Update ``wigToBigWig`` to kent-v365 version  in ``resolwebio/chipseq``
   Docker image
 - Change paths in HTML amplicon report template in ``resolwebio/dnaseq`` Docker
   image
+- Change Docker image in ``alignment-bowtie`` from ``resolwebio/legacy:1.0.0`` with
```

Comment:
Consider merging the changes below into a single entry since all processes share the old and the new Docker image. Are all those changes backward compatible?

Reply:
Changes are backward compatible, because there were't any changes in processors or in results, I haven't added or renamed any input or output.